AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet2Solutions.lean

```diff
@@ -0,0 +1,269 @@
+/-
+This is a solutions sheet.
+-/
+
+import Mathlib.Tactic
+
+/-Credit for some exercises goes to Kevin Buzzard and his Formalizing Mathematics
+course OR to the Natural Number Game. If you would like to learn more about Lean,
+Buzzard's course goes more in depth in relation to numerous undergraduate math topics.
```

Comment:
A link to Formalizing Mathematics 2024 would be helpful

Reply:
Done and also placed on Sheets 3-5